src/Stratis.Bitcoin.Tests/Utilities/DateTimeProviderTest.cs

```diff
@@ -10,9 +10,11 @@ public class DateTimeProviderTest
         [Fact]
         public void GetUtcNowReturnsCurrentUtcDateTime()
         {
+            int timeOffset = -96;
```

Comment:
Can you please create a new test in addition to this one and leave this one as is?